dnf-docker-test/features/updateinfo-2.feature

```diff
@@ -0,0 +1,172 @@
+        And I enable repository "base"
+        And I successfully run "dnf -y install TestA TestB"
+       Then rpmdb changes are
+         | State     | Packages         |
+         | installed | TestA/1, TestB/1 |
+
+  Scenario: updateinfo summary (when there's nothing to report)
+       When I successfully run "dnf updateinfo summary"
+       Then the command stdout should not match regexp "Updates Information Summary"
+
+  Scenario: updateinfo --summary (when there's nothing to report)
+       When I successfully run "dnf updateinfo --summary"
+       Then the command stdout should not match regexp "Updates Information Summary"
+
+  Scenario: updateinfo summary available (when there is an available update)
+       When I enable repository "ext1"
+        And I successfully run "dnf updateinfo summary available"
+       Then the command stdout should match regexp "Updates Information Summary: available"
+        And the command stdout should match regexp "1 Moderate Security notice"
+         
+  Scenario: updateinfo info
+       When I successfully run "dnf updateinfo info"
+       Then the command stdout should match regexp "TestB security update"
+        And the command stdout should match regexp "Update ID.*RHSA-2999:001"
+        And the command stdout should match regexp "Type.*security"
+        And the command stdout should match regexp "Updated"
+        And the command stdout should match regexp "Bugs.*BZ222"
+        And the command stdout should match regexp "CVEs.*CVE-2999-0001"
+        And the command stdout should match regexp "Description.*Security Advisory"
+        And the command stdout should match regexp "Severity.*Moderate"
+         
+  Scenario: updateinfo --info
+       When I successfully run "dnf updateinfo --info"
+       Then the command stdout should match regexp "TestB security update"
+        And the command stdout should match regexp "Update ID.*RHSA-2999:001"
+        And the command stdout should match regexp "Type.*security"
+        And the command stdout should match regexp "Updated"
+        And the command stdout should match regexp "Bugs.*BZ222"
+        And the command stdout should match regexp "CVEs.*CVE-2999-0001"
+        And the command stdout should match regexp "Description.*Security Advisory"
+        And the command stdout should match regexp "Severity.*Moderate"
+         
+  Scenario: updateinfo info bugfix (when there's nothing to report)
+       When I successfully run "dnf updateinfo info bugfix"
+       Then the command stdout should not match regexp "Update ID"
+
+  Scenario: updateinfo list
+       When I successfully run "dnf updateinfo list"
+       Then the command stdout should match regexp "RHSA-2999:001.*Moderate/Sec..*TestB-2-1"
+
+  Scenario: updateinfo --list
+       When I successfully run "dnf updateinfo --list"
+       Then the command stdout should match regexp "RHSA-2999:001.*Moderate/Sec..*TestB-2-1"
+
+#  Scenario: updateinfo list severity Moderate
```

Comment:
@glum23 Please can you explain this scenario, like why it is disabled and what is blocker? This is last blocker to merge it.